include/olt-manager.php

```diff
@@ -85,155 +60,43 @@ public static function instance() {
 	}
 
 	/**
-	 * Loads text domains
+	 * Loads textdomains.
 	 *
 	 * @since 0.1
 	 *
 	 * @return void
 	 */
 	public function load_textdomains() {
-		// Our load_textdomain_mofile filter has done its job. let's remove it before calling load_textdomain
-		remove_filter( 'load_textdomain_mofile', array( $this, 'load_textdomain_mofile' ) );
-		remove_filter( 'gettext', array( $this, 'gettext' ) );
-		remove_filter( 'gettext_with_context', array( $this, 'gettext_with_context' ) );
-		$new_locale = get_locale();
-
-		// Don't try to save time for en_US as some users have theme written in another language
-		// Now we can load all overridden text domains with the right language
-		if ( ! empty( $this->list_textdomains ) ) {
-			foreach ( $this->list_textdomains as $textdomain ) {
-				// Since WP 4.6, plugins translations are first loaded from wp-content/languages
-				if ( ! load_textdomain( $textdomain['domain'], str_replace( "{$this->default_locale}.mo", "$new_locale.mo", $textdomain['mo'] ) ) ) {
-					// Since WP 3.5 themes may store languages files in /wp-content/languages/themes
-					if ( ! load_textdomain( $textdomain['domain'], WP_LANG_DIR . "/themes/{$textdomain['domain']}-$new_locale.mo" ) ) {
-						// Since WP 3.7 plugins may store languages files in /wp-content/languages/plugins
-						load_textdomain( $textdomain['domain'], WP_LANG_DIR . "/plugins/{$textdomain['domain']}-$new_locale.mo" );
-					}
-				}
-			}
-		}
-
-		// First remove taxonomies and post_types labels that we don't need to translate
-		$taxonomies = get_taxonomies( array( '_pll' => true ) );
-		$post_types = get_post_types( array( '_pll' => true ) );
+		// Our load_textdomain_mofile filter has done its job. let's remove it before calling load_textdomain.
```

Comment:
We don't call `load_text_domain()` anymore.
```suggestion
		// Our load_textdomain_mofile filter has done its job. let's remove it to enable translation.
```